src/App.jsx

```diff
@@ -32,14 +49,16 @@ const App = () => {
           When pressing the button
         </p>
         <div className="answer-box">
-          {/* This is where the answer will be displayed */}
-            <NameCard />
+          {allNames.map(nameObj =>
+            <NameCard key={nameObj.id} id={nameObj.id} name={nameObj.name} deleteButton={handleDeleteOne} />
+          )}
```

Comment:
quedaría más limpio haciendo una destructuración del objeto 
```suggestion
          {allNames.map(({id, name})=>
            <NameCard key={id} id={id} name={name} deleteButton={handleDeleteOne} />
          )}
```